cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/StatefulSetOperator.java

```diff
@@ -54,30 +52,25 @@ public StatefulSetOperator(Vertx vertx, KubernetesClient client, long operationT
         this.operationTimeoutMs = operationTimeoutMs;
     }
 
+    public StatefulSetOperator(Vertx vertx, KubernetesClient client, long operationTimeoutMs, PodOperator podOperator) {
```

Comment:
Wouldn't it be better if one constructore called the other one? Now you are kind of duplicating the code.